modal/cli/secret.py

```diff
@@ -48,13 +48,14 @@ async def list_(env: Optional[str] = ENV_OPTION, json: bool = False):
 @synchronizer.create_blocking
 async def create(
     secret_name,
-    keyvalues: list[str] = typer.Argument(..., help="Space-separated KEY=VALUE items"),
+    keyvalues: Optional[list[str]] = typer.Argument(default=None, help="Space-separated KEY=VALUE items"),
     env: Optional[str] = ENV_OPTION,
-    force: bool = typer.Option(False, "--force", help="Overwrite the secret if it already exists."),
+    from_env: Optional[list[str]] = typer.Option(default=None, help="Name of environment variable holding the secret"),
```

Comment:
nit: potentially confusing that we have an `--env` flag specifying the destination *modal environment* whereas the `--from-env` specifies a variable name in the *local environment*

Naming is so hard :(

Reply:
Maybe `--from-env-var`? But (even more) verbose...

Reply:
FWIW [Vercel has similar challenges](https://vercel.com/docs/cli/env) 😄 

<img width="813" alt="image" src="https://github.com/user-attachments/assets/5a0f0ad1-edec-4dcb-9fea-c46b2d9322e4" />